slick.core.js

```diff
@@ -48,6 +60,36 @@
     this.isImmediatePropagationStopped = function () {
       return isImmediatePropagationStopped;
     };
+
+    this.getNativeEvent = function() {
+      return nativeEvent;
+    }
+
+    this.preventDefault = function() {
+      if(nativeEvent)
+        nativeEvent.preventDefault();
+      isDefaultPrevented = true;
+    }
+
+    this.isDefaultPrevented = function() {
+      if(nativeEvent)
```

Comment:
same comment as above to wrap in curly braces to avoid future bugs, there are a few of these in this file, they could be all be changed at the same time please